templates/store/publisher-details.html

```diff
@@ -0,0 +1,212 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}{% endblock %}
+
+{# TODO: title, meta, etc #}
+{% block meta_title %}{{ publisher }} in the Snap Store{% endblock %}
+{% block meta_description %}Install {{ publisher }} software for Linux in the Snap Store{% endblock %}
+
+{% if icon_url %}
+  {% block meta_image %}
+    {{ icon_url }}
+  {% endblock %}
+{% endif %}
+
+
+{% block content %}
+  <div class="p-strip--dark is-shallow" style="background: #262626">
+    <div class="row">
+      <div class="p-snap-heading">
+        {% if icon_url %}
+          <img class="p-snap-heading__icon" src="{{ icon_url }}" alt="{{ publisher }} snap" />
+        {% else %}
+          <img class="p-snap-heading__icon" src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" alt="" />
+        {% endif %}
+        <div class="p-snap-heading__title">
+          <h1 class="p-heading--two p-snap-heading__name">{{ publisher }}</h1>
+          <p class="p-snap-heading__publisher">
```

Comment:
```suggestion
          <p class="p-snap-heading__publisher u-no-margin--bottom">
```
Removes the extra blank space to balance the top and bottom of the strip a bit, imo.

Reply:
👍